docs/photutils/background.rst

```diff
@@ -158,13 +158,13 @@ mesh of a grid that covers the input data to create a low-resolution
 background map.  The final background or background rms map can then
 be generated by interpolating the low-resolution map.
```

Comment:
Yes, I didn't add the new Background classes to narrative docs yet.  I wanted to wait until the API was firmly decided.